python/ray/actor.py

```diff
@@ -215,6 +215,13 @@ def __init__(self):
         self.method_signatures = {}
         self.actor_method_num_return_vals = {}
         for method_name, method in self.actor_methods:
+            # Check if this method is static, and if so treat it like a class
+            # method by mocking the `__self__` attribute to look bound.
+            for cls in inspect.getmro(self.modified_class):
+                if method_name in cls.__dict__:
+                    if isinstance(cls.__dict__[method_name], staticmethod):
+                        method.__self__ = self.modified_class
```

Comment:
I came across a hard time trying to decide where to put this block of code. On one hand, it would make sense to go inside of `signature.extract_signature` or even `utils.is_class_method`, but the detection of the static method explicitly requires the class encapsulating the method (i.e. `self.modified_class` here). So I am not sure how much liberty I should take with throwing that structure around and exposing it to other modules.

Reply:
Hm, looks like https://stackoverflow.com/a/8727121/2213289 might work?

Reply:
Actually, ignore that - I think not... let me take a look around

Reply:
I think you can add a method like `is_static_method(method, class)`. This can be used in L230 (don't ignore_first if staticmethod), and also in L760 (`if is_class_method(method) or is_static_method(method, class)`).

Reply:
I was thinking of something like that too, but there are other places in the code that check if it is a class method and so just that change won't cut it. I'll try to take a look at changing all the code referencing this.

Reply:
That sounds good; I only found 2 places, but let me know if that's an oversight.

Reply:
I think you're right. I just made the change, let me know if there's anything else.